packages/google_maps_flutter/google_maps_flutter_web/lib/src/map_styler.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
I believe you'll need a

```dart
@JS()
library;
```

declaration here for https://github.com/dart-lang/sdk/issues/55039.

Reply:
Might also need it in whatever other files have an object literal factory in an extension type.